financial-connections/src/androidTest/java/com/stripe/android/financialconnections/FinancialConnectionsSheetActivityTest.kt

```diff
@@ -0,0 +1,17 @@
+package com.stripe.android.financialconnections
+
+import androidx.lifecycle.Lifecycle
+import androidx.test.core.app.ActivityScenario
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+
+internal class FinancialConnectionsSheetActivityTest {
```

Comment:
Could we move this to a unit test (via robolectric)? Seems unnecessary to run a test like this on an emulator.

Reply:
https://github.com/stripe/stripe-android/pull/5891/commits/3dc798de86946683233585752f85c0025d8cc9c8